src/components/TodoCard/TodoCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../../app/store';
+import { setSelectedTodo } from '../../features/currentTodo';
+
+type Props = {
+  todo: Todo;
+  showModalWindow: (show: boolean) => void;
+  isShowModal: boolean;
+};
+
+const TodoCard: React.FC<Props> = ({ todo, showModalWindow, isShowModal }) => {
+  const dispatch = useDispatch<AppDispatch>();
+  const selectedTodo = useSelector((state: RootState) => state.currentTodo);
+
+  const isSelectedTodo = selectedTodo?.id === todo.id;
+
+  const handleSelectTodo = (selectedElement: Todo) => {
+    dispatch(setSelectedTodo(selectedElement));
+  };
+
+  return (
+    <tr
+      data-cy="todo"
+      className={cn(
+        selectedTodo?.id === todo?.id && isShowModal
+          ? 'has-background-info-light'
+          : '',
+      )}
```

Comment:
are u using `cn` or ternary operator?) Change it or use ternary only
```suggestion
    className={cn({
      'has-background-info-light': selectedTodo?.id === todo?.id && isShowModal,
    })}
```